atlasdb-commons/src/main/java/com/palantir/common/base/BatchingVisitableView.java

```diff
@@ -47,7 +47,7 @@ private BatchingVisitableView() {
     @Override
     protected abstract BatchingVisitable<T> delegate();
 
-    public static <T> BatchingVisitableView<T> of(final BatchingVisitable<? extends T> underlyingVisitable) {
+    public static <T> BatchingVisitableView<T> of(final BatchingVisitable<T> underlyingVisitable) {
```

Comment:
We should document this as a breaking change (though it looks like a much smaller one)